### PR TITLE
libdrm: add option to build the nouveau and radeon driver

### PR DESCRIPTION
**Pull request by bkuhls**
Signed-off-by: Bernd Kuhls <bernd.kuhls@t-online.de>

Maintainer: @lucize 
Compile tested: x86_64

### CODE REVIEW

**Comment by lucize** [comment]
LGTM

**Comment by neheb** [comment]
Needs a PKG_RELEASE bump.

File: libs/libdrm/Makefile [hunk 1]
```diff
@@ -45,9 +47,9 @@ endef
 
 MESON_ARGS += \
 	$(if $(CONFIG_LIBDRM_INTEL),-Dintel=true -Dlibkms=true,-Dintel=false -Dlibkms=false) \
-	-Dradeon=false \
+	$(if $(CONFIG_LIBDRM_NOUVEAU),-Dnouveau=true,-Dnouveau=false) \
```
Comment (neheb) [inline libs/libdrm/Makefile:50]: Could change to -Dnouveau=$(if $(CONFIG_LIBDRM_NOUVEAU),true,false) . Although maybe that's too pedantic...

Comment (bkuhls) [reply libs/libdrm/Makefile]: done

**Comment by lucize** [comment]
you forgot PKG_RELEASE bump

**Comment by bkuhls** [comment]
> you forgot PKG_RELEASE bump

done

**Pull request merged**